DASH-IF-IOP.bs.md

```diff
@@ -10,9 +10,1070 @@ Live discussion in #document-authoring on Slack.
+
+In a dynamic MPD, a representation SHALL NOT contain unnecessary segment references except when any of the following applies, in which case an unnecessary segment reference MAY be present:
+
+1. The segment reference is for future content and will eventually become necessary.
+1. The segment reference is defined via [=indexed addressing=].
+1. The segment reference is defined by an `<S>` element that defines multiple references using `S@r`, some of which are necessary.
+1. Removal of the segment reference is not allowed by [[#timing-mpd-updates-remove-content|content removal constraints]].
+
+There SHALL NOT be gaps or overlapping [=media segments=] in a representation.
+
+Clients SHALL NOT present any samples from [=media segments=] that are entirely outside the [=period=], even if such [=media segments=] are referenced.
+
+<figure>
+	<img src="Images/Timing/SamplesOnPeriodBoundary.png" />
+	<figcaption>[=Media segments=] and samples need not align with [=period=] boundaries. Some samples may be entirely outside a [=period=] (marked gray) and some may overlap the [=period=] boundary (yellow).</figcaption>
+</figure>
+
+[=Media segment=] start/end points MAY be unaligned with [=period=] start/end points except when using [=simple addressing=], which requires the first [=media segment=] start point to match the [=period=] start point.
+
+Note: Despite constraints on first [=media segment=] alignment, [=simple addressing=] allows the media samples within the first [=media segment=] to be unaligned with the [=period=] start point, allowing for synchronization of different reprensetations.
+
+If a [=media segment=] overlaps a [=period=] boundary, clients SHOULD NOT present the samples that lie outside the [=period=] and SHOULD present the samples that lie either partially or entirely within the [=period=].
+
+Note: In the end, which samples are presented is entirely up to the client. It may sometimes be impractical to present [=media segments=] only partially, depending on the capabilties of the client platform, the type of media samples involved and any dependencies between samples.
+
+## Sample timeline ## {#timing-sampletimeline}
+
+<figure>
+	<img src="Images/Timing/TimelineAlignment.png" />
+	<figcaption>Sample timelines are mapped onto the [=MPD timeline=] based on parameters defined in the MPD.</figcaption>
+</figure>
+
+The samples within a [=representation=] exist on a linear <dfn>sample timeline</dfn> defined by the encoder that created the samples. One or more sample timelines are mapped onto the [=MPD timeline=] by metadata stored in or referenced by the MPD.
+
+Note: A sample timeline is linear - encoders are expected to use an appropriate timescale and sufficiently large timestamp fields to avoid any wrap-around. If wrap-around does occur, a new [=period=] must be started in order to establish a new sample timeline.
+
+The sample timeline is formed after applying any [[!ISOBMFF]] edit lists.
```

Comment:
From @haudiobe: This is called movie timeline or media time. Reference to CMAF/ISO BMFF

Reply:
That terminology is confusing as it is vague. Sample timeline was chosen in this contribution because it clearly outlines that this is the timing as specified on the sample level. "Movie" or "media" is too easily confused with MPD level timing.